setup.py

```diff
@@ -112,8 +112,14 @@ def cmake_generate(self):
             '-DOTIO_PYTHON_INSTALL:BOOL=ON',
             '-DOTIO_CXX_INSTALL:BOOL=ON',
             '-DCMAKE_BUILD_TYPE=' + self.build_config,
-            '-DPYBIND11_FINDPYTHON=ON',  # Smart tool to find python's libs
-        ]  # https://pybind11.readthedocs.io/en/latest/compiling.html#findpython-mode
+        ]
+
+        # Within the manylinux docker containers used for linux wheel builds
+        # pybind's findpython does not work for some reason
+        if not os.environ.get("CIBUILDWHEEL") or platform.system() != "Linux":
```

Comment:
@Tilix4 This looks like a similar issue as for Blender? I wonder if a similar conditional is necessary for some Blender?

Reply:
This is a conditional I came up with to maintain as much status quo as possible while only blocking my limited build case. I'd be open to input from the community if this flag should be treated as an allow or block list set of conditionals.

Reply:
I would strongly prefer not to put application specific branching in the setup.py if we can avoid it.

Reply:
@meshula Yes it looks like it. But I cannot ensure it is the exact same case.

Reply:
I'd happily address this by adding environment setup to the build to address it, but I couldn't find the environment magic that would appropriately hint the pybind magic. I'm open to looking more if someone with more familiarity with the pybind/cmake stuff has some hints about things to try.

Reply:
What if we check to see if os.environ['PYBIND11_FINDPYTHON'] is already set?  That way the user can disable it through an environment variable, which is really easy to do in GitHub actions:

`env PYBIND11_FINDPYTHON=OFF python -m pip install...` (assuming the environment variables propagate through.

Reply:
Oh, I like that idea better. I can use `CIBW_ENVIRONMENT` to push that setting down: https://cibuildwheel.readthedocs.io/en/stable/options/#environment

Reply:
I just tried with just the manylinux docker image locally and it's not finding python with the current code in this PR. So there is something funky.

Command I use:
```
docker run --rm -ti -e PLAT=manylinux2010_x86_64 -v $(pwd):/io quay.io/pypa/manylinux2010_x86_64 bash
[root@9a703412e29f io]# /opt/python/cp36-cp36m/bin/pip wheel . -v
```